### PR TITLE
Add quick instructions for building the Zip.

### PR DESCRIPTION
**Pull request by ryanwilson**
Added some instructions to build the zip. Will expand further with releasing information.

### CODE REVIEW

**Comment by maksymmalyhin** [comment]
@ryanwilson The description looks good and understandable, but I was not able to run `ZipBuilder` with no arguments. Once I started trying experimenting with arguments I realized I have no idea where I can get a template project or a "CoreDiagnostics.framework` file built with the Zip flag enabled". 

I think, adding a bit more context about such parameters to the README may save us from asking/answering the same questions many times.

File: ZipBuilder/README.md [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+## Running the Tool
+
+You can run the tool with `swift run ZipBuilder [ARGS]` or generate an Xcode project with `swift package generate-xcodeproj` and run within Xcode.
+
+## Launch Arguments
+
+See `main.swift` and the `LaunchArgs` struct for information on specific launch arguments.
+
+You can pass in launch arguments with Xcode by clicking "ZipBuilder" beside the Run/Stop buttons, clicking "Edit Scheme" and adding them in the "Arguments Passed On Launch" section. 
+
```
Comment (paulb777) [inline ZipBuilder/README.md]: Add typical options like:

+            argument = "-coreDiagnosticsDir  /Users/paulbeusterien/FirebaseCoreDiagnostics.framework -releasingSDKs /google/src/head/depot/google3/googlemac/iPhone/Firebase/Rapid/Releases/current.textproto -existingVersions /google/src/head/depot/google3/googlemac/iPhone/Firebase/Rapid/all_firebase_ios_sdks.textproto -cacheEnabled true -customSpecRepos sso://cpdc-internal/firebase -templateDir /Users/paulbeusterien/github/firebase-ios-sdk/ZipBuilder/Template"

Comment (ryanwilson) [reply ZipBuilder/README.md]: Great idea, added.

**Comment by ryanwilson** [comment]
Thanks for the feedback! Split it into 100 line length and added some typical arguments.

**Pull request merged**